pkg/lib/indexconfig/indexconfig.go

```diff
@@ -0,0 +1,124 @@
+package indexconfig
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/operator-framework/operator-registry/pkg/containertools"
+	"github.com/operator-framework/operator-registry/pkg/image"
+	"github.com/operator-framework/operator-registry/pkg/image/containerdregistry"
+	"github.com/operator-framework/operator-registry/pkg/image/execregistry"
+	"github.com/operator-framework/operator-registry/pkg/lib/certs"
+	registry "github.com/operator-framework/operator-registry/pkg/lib/registry"
+	regi "github.com/operator-framework/operator-registry/pkg/registry"
```

Comment:
Trying to think of a better name for this alias

Reply:
maybe call the `pkg/lib/registry` package `libregistry` and then call `regi` `registry` instead?